gear/identifier_lookup/src/python/identifier_app/main.py

```diff
@@ -82,7 +93,14 @@ def visit_row(self, row: Dict[str, Any], line_num: int) -> bool:
         Returns:
           True if there is a NACCID for the PTID, False otherwise
         """
-        writer = self.__get_writer()
+
+        if int(row.get(FieldNames.ADCID, -1)) != self.__adcid:
```

Comment:
would it be possible to use enrollment.enrollment_transfer.CenterValidator here?